pyrokinetics/miller.py

```diff
@@ -154,6 +161,8 @@ class Miller(LocalGeometry):
         Normalised Major radius (Rmajor/a_minor)
     Rgeo : Float
         Normalisd major radius of normalising field (Rreference/a)
+    Z0 : Float
```

Comment:
You've renamed to zmid below.

Reply:
Zmid is different. Z0 = Zmid/aminor

Reply:
I needed both as getting the regular flux surface requires Zmid. But when I do loop integrals around a flux surface to find q or Bunit I need Zmin/aminor = Z0. 

The distinction is especially important as a local input file may have Zmid/aminor but not Zmid.

Happy to rename if confusing?

Reply:
Think it's fine. I just saw you renaming from Z0 to Zmid at the same time you adding docs referring to Z0 so thought I should double check.